db/schema.rb

```diff
@@ -154,10 +154,10 @@
     t.datetime "updated_at",    null: false
   end
 
-  add_index "crosswalks", ["cross"], name: "index_crosswalks_on_cross", using: :btree
+  add_index "crosswalks", ["cross"], name: "index_crosswalks_on_cross", unique: true, using: :btree
```

Comment:
We should take the unique constraint off the index - I have found cases where crosswalk information is not unique